api_EPA.py

```diff
@@ -23,12 +25,27 @@
 filename = "test.csv"
 file_path = os.path.join("./", filename)
 
+#case where the script is run twice back to back and the second time there aren't any results to show
+#if array length is 0 exit // no new data
+# end date at most 2 days later that the start time
 
 def main():
-    # TODO: get last_updated timestamp, fetch all data since
-    print("Fetching EPA data")
 
-    epa_data = pull_from_epa_api(start_time="2020-07-06T00", end_time = "2020-07-07T00")
+    #raw_end_time = strftime('%Y-%m-%dT%H:%M', gmtime())
+
+    start_time = get_start_time()
+
+    temp = get_start_time()
+
+    starting_day = int(temp[9])
+
+    end_day = starting_day + 2
```

Comment:
What if the starting_day is on the 31st of the month? Instead of working with strings, I'd recommend using native python datetime objects and methods (https://docs.python.org/3/library/datetime.html). 

I'm sure there's a way to explicitly add 2 days to a previous dateTime. In javascript all data manipulation is done in milliseconds, so adding 2 days would mean adding `1000 * 60 * 60 * 24 * 2` milliseconds.